CHANGES.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+### 1.81 - 2021-05-01
+
+##### Additions :tada:
+
+- Updated [gltf-pipeline](https://github.com/CesiumGS/gltf-pipeline) to 3.0.2. [#9477](https://github.com/CesiumGS/cesium/pull/9477)
```

Comment:
No need to include this change in `CHANGES.md`. There aren't any user facing changes as part of this PR.

Reply:
Just to double check, I was basing this off of https://github.com/CesiumGS/cesium/pull/6805/files. Should I still remove the CHANGES.md changes here?

Reply:
Ah, that older change was probably unnecessary as well. You should still remove the CHANGES.md entry here.